app/services/phone_country_codes.rb

```diff
@@ -0,0 +1,3 @@
+class PhoneCountryCodes
+  US_AND_US_TERRITORY_CODES = ['AS', 'GU', 'MP', 'PR', 'US', 'VI']
+end
```

Comment:
I wish we had a better home for this constant, but I can't think of one... `PhoneNumberCapabilities` maybe since it has `INTERNATIONAL_CODES`?

Reply:
I think I'm going to convert this into a config item? `PhoneNumberCapabilities` makes sense to me too

Reply:
Config makes sense to me too, maybe even generalized to “supported IDV address verification country codes”.

Reply:
renamed and made configurable in 37b286e206a2bf7c2734335c84671150062c3105